src/main/java/ch/njol/skript/effects/EffBroadcast.java

```diff
@@ -76,33 +70,17 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelaye
 	}
 	
 	@Override
-	@SuppressWarnings("deprecation")
-	public void execute(Event e) {
+	public void execute(Event event) {
 		List<CommandSender> receivers = new ArrayList<>();
 		if (worlds == null) {
 			receivers.addAll(Bukkit.getOnlinePlayers());
 			receivers.add(Bukkit.getConsoleSender());
 		} else {
-			for (World world : worlds.getArray(e))
+			for (World world : worlds.getArray(event))
 				receivers.addAll(world.getPlayers());
 		}
 
-		for (Expression<?> message : getMessages()) {
-			if (message instanceof VariableString) {
-				BaseComponent[] components = BungeeConverter.convert(((VariableString) message).getMessageComponents(e));
```

Comment:
What's the point of BungeeConverter when VariableString also stores the MessageComponents? Can't BungeeConverter just be removed entirely if we're not going to use it.

Reply:
MessageComponent is a custom Skript type. BungeeConverter converters our MessageComponent into Spigot/Bungee's component format

Reply:
So why was the old code you deleted in this pull request not used again with the BungeeConverter?